infrastructures/docker/src/main/java/org/eclipse/che/workspace/infrastructure/docker/DockerRuntimeContext.java

```diff
@@ -35,32 +42,52 @@
  * Docker specific implementation of {@link RuntimeContext}.
  *
  * @author Alexander Garagatyi
+ * @author Yevhenii Voievodin
  */
 public class DockerRuntimeContext extends RuntimeContext {
-    private final DockerEnvironment    dockerEnvironment;
-    private final DockerRuntimeFactory dockerRuntimeFactory;
-    private final List<String>         orderedContainers;
-    private final String               devMachineName;
-    private final String               apiEndpoint;
 
-    @Inject
+    private static final Logger LOG = LoggerFactory.getLogger(DockerRuntimeContext.class);
+
+    private final DockerEnvironment         dockerEnvironment;
+    private final List<String>              orderedContainers;
+    private final String                    devMachineName;
+    private final String                    apiEndpoint;
+    private final DockerRuntimeFactory      runtimeFactory;
+    private final DockerContainers          containers;
+    private final RuntimeConsistencyChecker consistencyChecker;
+    private final WorkspaceSharedPool       sharedPool;
```

Comment:
For some reasons i though that docker infra is in workspace module, i will create a separate pool for docker infra, thank you!